### PR TITLE
Feat: Added Static Signup Page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added Signup-Page in Trendy-t E-Commerce.
Signup

### CODE REVIEW

File: src/Pages/Signup-Page/signup.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+/* signup form ends here */
+
+/*** Scroll bar ***/
+
+::-webkit-scrollbar {
+  width: 5px;
+  height: 5px;
+}
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 10px rgb(255, 255, 255);
```
Comment (kedark152) [inline src/Pages/Signup-Page/signup.css:112]: You can create a variable for box-shadow value

**Review by kedark152 — COMMENTED** [review]
Code is written well. UI Design looks Nice.
But I am unable to access Live Links https://deploy-preview-11--trendyt-e-commerce.netlify.app/login/ & https://deploy-preview-11--trendyt-e-commerce.netlify.app/signup.
Please check if you can make it live.

File: src/Pages/Signup-Page/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+import "./signup-Responsive.css";
+
+export function Signup() {
+  return (
+    <div className="signup-page">
+      <Nav />
+      <NavForMobile />
+      <main className="signup__main">
+        <section className="signup">
+          <div className="signup--header">
```
Comment (ShreyashMahajan) [inline src/Pages/Signup-Page/Signup.jsx:14]: here you can write signup__header

File: src/Pages/Signup-Page/Signup.jsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                <label
+                  className="signup-form__label signup-form--lastname"
+                  htmlFor="userName"
+                >
+                  <b>SurName</b>
+                </label>
+                <input
+                  className="signup-form__input-demo signup-form__input--username"
+                  type="text"
+                  placeholder="Surname"
```
Comment (ShreyashMahajan) [inline src/Pages/Signup-Page/Signup.jsx:43]: *sirname

**Review by ShreyashMahajan — APPROVED** [review]
Overall code looks well structured and good to merge

**Pull request merged**